build/build.cake

```diff
@@ -20,7 +20,7 @@ var target = Argument("target", "Default");
 // VERSIONS
 //////////////////////////////////////////////////////////////////////
 
-var gitVersioningVersion = "2.1.65";
+var gitVersioningVersion = "3.1.91";
 var inheritDocVersion = "1.1.1.1";
```

Comment:
Would it make sense to update InheritDoc on this PR?

Reply:
Sure, to the latest in v1 or do we see if v2 breaks anything?

Reply:
Try v2 🐱‍💻

Reply:
Will do, didn't see any call out to breaking changes.